isodatetime/tests.py

```diff
@@ -1204,6 +1204,32 @@ def test_timepoint_time_zone(self):
                             duration, data.Duration(days=0),
                             i_test_date_str + " - " + j_test_date_str)
 
+    def test_timepoint_adding_truncated_timepoint(self):
+        """See issue #80. When adding a truncated time, the result timepoint
+        must not be inconsistent, e.g.
+
+        2000-01-01T00:00:00Z + 03-30T00:00:00Z = 2000-03-02T00:00:00Z
```

Comment:
It isn't. I included this as an example of an error, with the expected correct behaviour in the line that follows (bug: was 2000-03-01T00:00:00Z, but expected 2000-03-30T00:00:00Z). But now that you mentioned it, I think this can be confusing, and not really helpful to first see a bad example, so replacing it by just the expected value :+1: